src/libraries/Microsoft.Extensions.Logging.Console/src/Microsoft.Extensions.Logging.Console.csproj

```diff
@@ -57,7 +57,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <NETStandardCompatError Include="netcoreapp2.0" Supported="netcoreapp3.1" />
+    <NETStandardCompatError Include="netcoreapp2.0" Supported="$(NetCoreAppMinimum)" />
```

Comment:
Would it make sense to default NETStandardCompatError/@Supported to `$(NetCoreAppMinimum)`?

Reply:
I do not know why I am here. But I love you.

Reply:
Even though we mostly care about .NETCoreApp, as .NETStandard has multiple compatible frameworks, I wouldn't want to default it to .NETCoreApp. As an example, for NET7 the min .NETFramework version will be net462 and we will make use of this infrastructure for .NETFramework as well.

Reply:
Defaulting could happen in Targets after the project sets the Include value.  You could handle both cases based on TFI of the include.  It's just a suggestion, but could avoid boilerplate if we think the boilerplate isn't adding value to the project.  I guess from that perspective we could automatically calculate these entirely since we see what the project is targeting.

Reply:
> I guess from that perspective we could automatically calculate these entirely since we see what the project is targeting.

Tbh that's what I would prefer. When I added this infrastructure in a previous PR I considered automation but didn't do it for different reasons. That said, I definitely wouldn't want to clean this up as part of this PR. For now I'm fine with having these items in the project file.

Reply:
Yeah, also fine with it for now.  Just had this gut-reaction when I saw this being treated like boilerplate.  If we do end up keeping it in the project file then we should add it to the docs: https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/libraries-packaging.md